src/content/docs/change-tracking/change-tracking-cli.mdx

```diff
@@ -64,35 +71,9 @@ newrelic entity deployment create --guid INSERT_YOUR_GUID_HERE --version <1.0.0>
     --trace            trace level logging
 ```
 
-## List the changes you are tracking [#create-list]
-
-Once you've designated some changes you want to track, you can list them by running a GraphQL query in NerdGraph or executing a NRQL query in the New Relic UI.
-
-### GraphQL query [#list-with-graphql]
-
-You can execute a query similar to this in [NerdGraph](https://api.newrelic.com/graphiql), our GraphiQL IDE:
-
-```
-newrelic  --profile xyz nerdgraph query '{ actor { entity(guid: "MTE2NjAxMzJ8QVBNfEFQUExJQ0FUSU9OfDg0MTY1OTI") { deploymentSearch { results { deploymentId } } guid name } } }'
-{
-  "actor": {
-    "entity": {
-      "deploymentSearch": {
-        "results": [
-          {
-            "deploymentId": "445784"
-          }
-        ]
-      },
-      "guid": "MTE2NjAxMzJ8QVBNfEFQUExJQ0FUSU9OfDg0MTY1OTI",
-      "name": "deployment-marker-api (production)"
-    }
-  }
-}
-```
-
-Alternatively, you can execute a [NRQL query](/docs/change-tracking/change-tracking-graphql#key-signals) against the New Relic database to retrieve a list of those changes.
-
 ## What's next [#whats-next]
 
-After you've designated the changes you want to track, you can analyze the effect of those changes in the New Relic UI. For details, see [How to view and analyze your changes in New Relic](/docs/change-tracking/change-tracking-view-analyze).
\ No newline at end of file
+After you've used the CLI to designate the changes you want to track, you can analyze the effect of those changes in the New Relic UI:
```

Comment:
Designate feels oddly formal. Why not "choose" or "pick"?

Reply:
@paperclypse I agree that "designate" seems formal. This isn't just a simple matter of selecting or choosing something in the UI. You have to run a GraphQL mutation that tells New Relic what deployments you want to track. Or, you run a command in the CLI that creates a GraphQl mutation for you. Perhaps "specify" is an option, but at this point, I'm going to keep "designate" since this is a complicated process.